catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -263,6 +263,26 @@
             <groupId>com.vividsolutions</groupId>
             <artifactId>jts</artifactId>
         </dependency>
+
+        <!-- Cyclops Dependencies
+        [root] com.oath.cyclops:cyclops:[10.0.0,11.0.0)
+         |-> org.reactivestreams:reactive-streams:[1.0.0,2.0.0)
+         |-> org.agrona:agrona:[0.9.0,1.0.0) -->
+        <dependency>
+            <groupId>com.oath.cyclops</groupId>
+            <artifactId>cyclops</artifactId>
+            <version>10.0.0-M1</version>
```

Comment:
✏️ Generally we try to avoid milestone releases.  It does not look like Cyclops has an official release yet and is under very active milestone releasing.

https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.oath.cyclops%22%20AND%20a%3A%22cyclops%22